File: GGAndroid_2.0/GGScanner/scanner-sample/src/main/java/io/scanner/sample/CreateCodeSampleActivity.java
package io.scanner.sample;

import io.ganguo.vmodel.ViewModelActivity;
import io.ganguo.viewmodel.databinding.IncludeHfRecyclerBinding;
import io.scanner.sample.viewmodel.CreateCodeSampleVModel;

/**
 * <p>
 * 生成二维码&&条形码Demo
 * </p>
 * Created by leo on 2018/8/9.
 */
public class CreateCodeSampleActivity extends ViewModelActivity<IncludeHfRecyclerBinding, CreateCodeSampleVModel> {
    @Override
    public CreateCodeSampleVModel createViewModel() {
        return new CreateCodeSampleVModel();
    }

    @Override
    public void onViewAttached(CreateCodeSampleVModel viewModel) {

    }
}

File: GGAndroid_3.0/main/app-scanner-sample/src/main/java/io/scanner/sample/CreateCodeSampleActivity.kt
package io.scanner.sample

import io.ganguo.viewmodel.core.BaseViewModel
import io.ganguo.viewmodel.core.view.ViewModelActivity
import io.ganguo.viewmodel.databinding.IncludeHfRecyclerBinding
import io.scanner.sample.viewmodel.CreateCodeSampleVModel

/**
 * <pre>
 * author : leo
 * time   : 2018/10/20
 * desc   : 生成二维码&&条形码Demo
</pre> *
 */
class CreateCodeSampleActivity : ViewModelActivity<IncludeHfRecyclerBinding, CreateCodeSampleVModel>() {
    override fun createViewModel(): CreateCodeSampleVModel {
        return CreateCodeSampleVModel()
    }

    override fun onViewAttached(viewModel: BaseViewModel<Any>) {

    }
}
